packages/wonder-blocks-dropdown/src/components/select-opener.tsx

```diff
@@ -229,44 +225,58 @@ const _generateStyles = (
 
     let newStyles: Record<string, any> = {};
     if (light) {
+        const focusHoverStyling = {
+            borderColor: error ? tokens.color.red : tokens.color.white,
+            borderWidth: tokens.spacing.xxxxSmall_2,
+            paddingLeft: adjustedPaddingLeft,
+            paddingRight: adjustedPaddingRight,
+        };
         newStyles = {
             default: {
                 background: error ? tokens.color.fadedRed8 : "transparent",
-                color: placeholder ? tokens.color.white50 : tokens.color.white,
+                color: error
+                    ? tokens.color.offBlack64
+                    : placeholder
+                    ? tokens.color.white50
+                    : tokens.color.white,
                 borderColor: error ? tokens.color.red : tokens.color.white50,
                 borderWidth: tokens.border.width.hairline,
-            },
-            focus: {
-                borderColor: error
-                    ? tokens.color.fadedRed8
-                    : tokens.color.white,
-                borderWidth: tokens.spacing.xxxxSmall_2,
-                paddingLeft: adjustedPaddingLeft,
-                paddingRight: adjustedPaddingRight,
-            },
-            active: {
-                paddingLeft: adjustedPaddingLeft,
-                paddingRight: adjustedPaddingRight,
-                borderColor: error ? tokens.color.red : tokens.color.fadedBlue,
-                borderWidth: tokens.border.width.thin,
-                color: placeholder
-                    ? mix(tokens.color.white32, tokens.color.blue)
-                    : tokens.color.fadedBlue,
-                backgroundColor: error
-                    ? tokens.color.fadedRed
-                    : tokens.color.activeBlue,
+                ":hover:not([aria-disabled=true])": focusHoverStyling,
+                ":focus-visible:not([aria-disabled=true])": focusHoverStyling,
```

Comment:
I believe we can use `:focus-visible` now ([caniuse](https://caniuse.com/?search=focus-visible)) without fallback `:focus` styling (like this [example of fallback focus styling](https://github.com/Khan/wonder-blocks/blob/main/packages/wonder-blocks-icon-button/src/components/icon-button-core.tsx#L355-L360)) since the [minimum Safari version](https://khanacademy.atlassian.net/wiki/spaces/ENG/pages/103612417/Browser+Support) we support now is 15.6.

Let me know otherwise though!